src/houston/public/styles/houston.css

```diff
@@ -8,37 +8,28 @@
 ******/
 
 nav {
-    background-color: #3a416f;
+    background-color: #403757;
```

Comment:
on the actual website the navbar has no background color, we should do the same here for the main entry page